src/main/java/com/synopsys/integration/alert/channel/email/descriptor/EmailDistributionUIConfig.java

```diff
@@ -45,11 +50,13 @@
     private static final String LABEL_ADDITIONAL_ADDRESSES = "Additional Email Addresses";
     private static final String LABEL_ADDITIONAL_ADDRESSES_ONLY = "Additional Email Addresses Only";
     private static final String LABEL_PROJECT_OWNER_ONLY = "Project Owner Only";
+    private static final String LABEL_ATTACHMENT_FORMAT = "Attachment File Type";
 
-    private static final String EMAIL_SUBJECT_LINE_DESCRIPTION = "The subject line to use in the emails sent for this distribution job.";
+    private static final String DESCRIPTION_EMAIL_SUBJECT_LINE = "The subject line to use in the emails sent for this distribution job.";
     private static final String DESCRIPTION_ADDITIONAL_ADDRESSES = "Any additional email addresses (for valid users of the provider) that notifications from this job should be sent to.";
     private static final String DESCRIPTION_ADDITIONAL_ADDRESSES_ONLY = "Rather than sending emails to users assigned to the configured projects, send emails to only the users selected in 'Additional Email Addresses'.";
-    private static final String EMAIL_PROJECT_OWNER_ONLY_DESCRIPTION = "If true, emails will only be sent to the administrator(s) of the project. Otherwise, all users assigned to the project will get an email.";
+    private static final String DESCRIPTION_EMAIL_PROJECT_OWNER_ONLY = "If true, emails will only be sent to the administrator(s) of the project. Otherwise, all users assigned to the project will get an email.";
+    private static final String DESCRIPTION_ATTACHMENT_FORMAT = "If a file type is selected, a file of that type, representing the message content, will be attached to the email.";
```

Comment:
I feel like we should remove the second comma here. It's a little weird reading it.